common/app/model/dotcomrendering/DotcomRenderingDataModel.scala

```diff
@@ -175,14 +177,15 @@ object DotcomRenderingDataModel {
       pageType: PageType,
   ): DotcomRenderingDataModel = {
     apply(
-      page,
-      request,
-      None,
+      page = page,
+      request = request,
+      pagination = None,
       linkedData = Nil, // TODO
       mainBlock = blocks.main,
       bodyBlocks = blocks.body.getOrElse(Nil),
-      pageType,
-      page.related.hasStoryPackage,
+      pageType = pageType,
+      hasStoryPackage = page.related.hasStoryPackage,
```

Comment:
I assume these changes are either boyscouting or just the IDE having ideas?

Reply:
the former yes